### ISSUE TITLE
PATH in docker image is not set correctly

### ISSUE THREAD

**Issue by TheisFerre** [bentoml/bentoctl#159]
Hi

I am trying to use Bentoml, bentoctl and the aws-sagemaker-deploy operator to deploy a simple sklearn model. The sklearn model is tracked in a MLFlow experiment where i am able to load it and from there build a bento.

When i have build the bento, i want to use bentoctl to create a docker image that is ready to use in AWS Sagemaker, using the aws-sagemaker-deploy operator. However, i don't want to create a new ECR repository or use the terraform template for the infrastructure related to the sagemaker endpoint. Because of that, i run the bentoctl with the --dry-run flag
`
bentoctl build ... --dry-run
`
This works and my container is build successfully. 
![image](https://user-images.githubusercontent.com/37203334/169102435-f2521988-e4f6-44b5-9054-24d4f65ca7b0.png)

After it is built, i push the container to my own ECR repository. To test that the container is compatible with Sagemaker, i created the endpoint-configuration, sagemaker model and endpoint myself. However, when i create the endpoint i get an error. 

![image](https://user-images.githubusercontent.com/37203334/169102739-084027ee-d190-442e-85fb-1071d1bcb29b.png)

The error is related the the `PATH`, so i pulled the image and started it locally. Here i saw that the `/home/bento` folder was indeed missing from the `PATH`.
![image](https://user-images.githubusercontent.com/37203334/169103398-81d86d73-cb56-4c5d-8b62-244eaef6741e.png)

Can you help me figure out why this is the case? In the aws-sagemaker-deploy operater, the last layer of the Docker template is `ENV PATH=$BENTO_PATH:$PATH`, but it does not seem to work.

**Comment by TheisFerre** [comment]
So i managed to figure out what the problem is. As mentioned in the issue, it seems that the Dockerfile.Template of the aws-sagemaker-deploy operator does not add the`BENTO_PATH` to the `PATH`. I.e. this issue is probably more related to the operator.

To make it work, i added the following.
`
RUN echo "export PATH=$BENTO_PATH:$PATH" >> /root/.bashrc
`

I am not sure why the existing layer in the Dockerfile is not adding the folder to the path though.

**Comment by jjmachan** [comment]
@TheisFerre thank you for raising this issue but unfortunately, I still can't reproduce this. If you have joined our slack group can you share your slack ID and I'll reach out to you over there?

If not do you mind sharing `docker version` and `bentoctl operator version`

**Comment by yubozhao** [comment]
@TheisFerre Hello, just want to check in and see where the progress is.  Did you get it working?

**Comment by TheisFerre** [comment]
Thanks for checking in @yubozhao! 

I have been in a dialogue with @jjmachan on Slack. We found that the problem seems to be when building the image with Github Actions, as I had no issues when doing it locally. I did find a work-around, so the issue is not blocking me in anyway right now.

I can close the issue here and create a new one in the aws-sagemaker-deploy repo, where i can provide a more detailed description.
